turbinia/output_manager.py

```diff
@@ -266,6 +266,9 @@ class GCSOutputWriter(OutputWriter):
     client (google.cloud.storage.Client): GCS Client
   """
 
+  # 10MB by default
+  CHUNK_SIZE = 10 * (2 ** 20)
+
```

Comment:
Is there any documentation around this? Why is this the optimal chunk size?

Reply:
So, I can't seem to find it now, but originally I found references to using 10MB chunk size.  I can't seem to find that reference now though, so I filed #161 to see if we can find a better default value.